t/build_man.t

```diff
@@ -44,10 +44,15 @@ ok((my $stdout = tie *STDOUT, 'TieOut'), 'tie stdout');
 
 {
     local $Config{installman3dir} = File::Spec->catdir(qw(t lib));
-    my $mm = WriteMakefile(
-        NAME            => 'Big::Dummy',
-        VERSION_FROM    => 'lib/Big/Dummy.pm',
-    );
+    my $mm;
```

Comment:
Why is this commit needed? Where is README.pod coming from?

Reply:
It's needed to suppress the given warning that occurs in all MSWin32 tests ... see <https://ci.appveyor.com/project/rivy/perl-extutils-makemaker/builds/20544004>.

As a test, it a harmless change that cleans up the output and suppresses the spurious WARNINGS. I don't see the WARNING or file locally when tested. I can gate is specifically to AppVeyor if you see some problem with it.

Reply:
I re-tested it without the suppression, and it occurs locally as well.
There is no *README.pod* file to interfere, just the plain README file. It just spits out the spurious warnings. I'm don't know why it occurs on MSWin32, but not *nix.

Reply:
I think this PR should not concern itself with these warnings, and if there is a bug with them on Win32 that should be fixed separately.

Reply:
Ok, so the "README.pod" file is left over from the creation on [line 41](https://github.com/Perl-Toolchain-Gang/ExtUtils-MakeMaker/pull/332/files#diff-4473c5dc5a51f147c095c02ce3b60062L41).

There is no associated test, so I likely assumed at the time that it was needed for something and didn't touch it. 

Placing a following `unlink $README or die ...` does fix the problem, and, much more directly, just removing [lines 40-41](https://github.com/Perl-Toolchain-Gang/ExtUtils-MakeMaker/pull/332/files#diff-4473c5dc5a51f147c095c02ce3b60062L40-41) completely fixes it as well.

I don't know why it's not a WARNING on *nix, just on Windows. And I don't know really why it's there to begin with, so I'm hesitant to just remove it. Although the tests run fine with it completely removed (for both Windows and *nix).

But, I do think that it needs to be fixed, either with the change I have here, the `unlink ...`, or complete removal of those two lines, as the WARNINGS are a useless distraction while testing in Windows.